ah @nsmith- I just built HEAD and I see this error there as well... It looks like it is coming from broken python (?), or how to guess from where is coming missing libarchive dependency..

```
#5 159.3 Remove all contents from the following package caches?
#5 159.3   - /opt/conda/pkgs
#5 159.7 Traceback (most recent call last):
#5 159.7   File "/opt/conda/bin/conda-build", line 7, in <module>
#5 159.7     from conda_build.cli.main_build import main
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/conda_build/cli/main_build.py", line 19, in <module>
#5 159.7     import conda_build.api as api
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 22, in <module>
#5 159.7     from conda_build.config import (Config, get_or_merge_config, get_channel_urls,
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/conda_build/config.py", line 18, in <module>
#5 159.7     from .variants import get_default_variant
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/conda_build/variants.py", line 18, in <module>
#5 159.7     from conda_build.utils import ensure_list, get_logger, islist, on_win, trim_empty_keys
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/conda_build/utils.py", line 9, in <module>
#5 159.7     import libarchive
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/libarchive/__init__.py", line 1, in <module>
#5 159.7     from .entry import ArchiveEntry
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/libarchive/entry.py", line 6, in <module>
#5 159.7     from . import ffi
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/libarchive/ffi.py", line 108, in <module>
#5 159.7     errno = ffi('errno', [c_archive_p], c_int)
#5 159.7   File "/opt/conda/lib/python3.8/site-packages/libarchive/ffi.py", line 95, in ffi
#5 159.7     f = getattr(libarchive, 'archive_'+name)
#5 159.7   File "/opt/conda/lib/python3.8/ctypes/__init__.py", line 394, in __getattr__
#5 159.7     func = self.__getitem__(name)
#5 159.7   File "/opt/conda/lib/python3.8/ctypes/__init__.py", line 399, in __getitem__
#5 159.7     func = self._FuncPtr((name_or_ordinal, self))
#5 159.7 AttributeError: /opt/conda/bin/python: undefined symbol: archive_errno
```

--- a/base/Dockerfile
+++ b/base/Dockerfile
@@ -1,5 +1,9 @@
 FROM continuumio/miniconda3:latest
 
+RUN apt-get update \
+     && apt-get install -yq --no-install-recommends libarchive-dev \
+     && apt-get clean && rm -rf /var/lib/apt/lists/*
+
 # XRootD + CA + VOMS + Coffea
 RUN conda install --yes --freeze-installed \
       -c conda-forge \

